bokehjs/src/coffee/models/mappers/linear_color_mapper.coffee

```diff
@@ -22,6 +22,9 @@ class LinearColorMapper extends ColorMapper.Model
       @_reserve_color = parseInt(@get('reserve_color').slice(1), 16)
       @_reserve_val   = @get('reserve_val')
 
+    @listenTo(this, 'change', () ->
```

Comment:
This needs to be factored out onto the parent ColorMapper class, along with some duplicate properties.